src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -206,6 +197,30 @@ foam.CLASS({
           }));
         }
       }
+    },
+    {
+      name: 'footer',
+      label: 'Create an account',
+      section: 'footerSection',
+      buttonStyle: 'LINK',
+      code: function(X) {
+        window.history.replaceState(null, null, window.location.origin);
```

Comment:
Generally, better to do X.window or import 'window' and do this.window. Accessing the global window prevents the code from working with multiple windows.